Package.swift

```diff
@@ -0,0 +1,22 @@
+// swift-tools-version:4.2
+
+import PackageDescription
+
+let package = Package(
+    name: "ReSwift-Thunk",
+    products: [
+      .executable(name: "ReSwift-Thunk", targets: ["ReSwift-Thunk"])
```

Comment:
Should this be `library`, rather than `executable`?

Reply:
Ah yes, it should. Good eye—I used [ReSwift-Router](https://github.com/ReSwift/ReSwift-Router/blob/master/Package.swift) as a template, I think that should probably be switched to `library` too.